newIDE/app/src/MainFrame/index.js

```diff
@@ -541,6 +552,19 @@ const MainFrame = (props: Props) => {
     onOpenNewProjectSetupDialog: () => setNewProjectSetupDialogOpen(true),
   });
 
+  const {
+    games,
+    gamesFetchingError,
+    fetchGames,
+    onGameUpdated,
+  } = useGamesList();
```

Comment:
I've moved this into the mainframe to pass it down, I wanted at first to do the captures logic automatically inside this, but ended up struggling to prevent an effect trigger too many times.
I was that close to create a GamesContext because it's starting to be a mess to pass down games etc everywhere

Reply:
See another comment, this could be a `gamesList: GamesList` type?